src/strategy/strategy-investigator.class.ts

```diff
@@ -27,6 +27,12 @@ export class StrategyInvestigator {
 
   private getStrategy(request: Request): ChainableStrategy {
     const url = getRequestUrl(request)
+    const hasTrailingSlash = url.href.endsWith("/")
```

Comment:
```suggestion
    const hasTrailingSlash = url.pathname.endsWith("/")
```

`href` includes the query string, so you would miss calls to `/test/?query=true` with this approach.

Reply:
I did this on purpose, but I guess you're right. Will change it.

Reply:
@LuudJanssen Please have another look